package.json

```diff
@@ -8,16 +8,16 @@
   },
   "bugs": "https://github.com/TryGhost/Ghost/issues",
   "private": true,
-  "version": "0.11.3",
+  "version": "0.11.8",
   "dependencies": {
-    "casper": "TryGhost/Casper#1.3.4",
-    "ghost": "0.11.3",
+    "casper": "TryGhost/Casper#1.3.6",
+    "ghost": "0.11.8",
     "ghost-s3-storage-adapter": "3.0.4",
     "ncp": "^2.0.0",
     "pg": "latest"
   },
   "engines": {
-    "node": "~0.12.0"
+    "node": "^6.10.3"
```

Comment:
Hmm, as I noted in the other PR, it doesn't change anything as Heroku chooses node v6.10.3 in both cases, but yeah just to be safe I'll revert to the Ghost version, it'll we clearer for future updates.